index.js

```diff
@@ -20,7 +20,7 @@ var matchHtmlRegExp = /["'&<>]/
  * @public
  */
 
-module.exports = escapeHtml
+module.exports = { escapeHtml, escapeHtmlFast, escapeHtmlNoRegex }
```

Comment:
Please make sure your syntax works on the Node.js versions declared in our `package.json`; generally ES5. If there are now multiple exports, they of course should all be documented on the README.

I also see there is no longer any default export, definately making this a major version bump, at least.

Reply:
That's why I said:

> If we agree on a path forward I can of course clean up the PR to remove the extra code, etc. 

_I pushed what I had so that anyone could easily test it exactly as I had._ (see both results side by side, tweak both, etc)  Unless we _decide_ to change the API these changes don't need to happen.  I'm just first trying to get agreement on what we're doing here.

> generally ES5.

Ah, I'm spoiled by our ES6 codebase, but I can fix. (or again, simply revert)

> I also see there is no longer any default export, definately making this a major version bump, at least.

I'd agree _if_ we are going to have multiple functions.